### PR TITLE
Do not run tests not suited for the current Plone version

### PR DESCRIPTION
**Pull request by jensens**
this is a trial/idea how to solve the problem with tests for older plone versions. Have a look and comment please.

### CODE REVIEW

**Comment by jensens** [comment]
@tisto you've been interested in this.

**Comment by jensens** [comment]
this solves also the failing tests in plone.app.upgrade here:
http://jenkins.plone.org/job/plone-5.0-python-2.7/4064/#showFailuresLink

File: plone/app/upgrade/utils.py [hunk 1]
```diff
@@ -11,10 +11,18 @@
...
+
 _marker = []
 
 logger = logging.getLogger('plone.app.upgrade')
 
+plone_version = pkg_resources.get_distribution("Products.CMFPlone").version
+
+
+def version_match(target):
+    return target <= plone_version < (target + 'zzzzzzz')
```
Comment (thet) [inline plone/app/upgrade/utils.py]: there's another way of comparing versions - which is maybe more robust:
```
>>> from pkg_resources import parse_version
>>> parse_version('1.0') < parse_version('1.1.dev')
... < parse_version('1.1.a1') < parse_version('1.1.a2')
... < parse_version('1.1.b') < parse_version('1.1.rc1')
... < parse_version('1.1') < parse_version('1.1.1')
```
i don't know, if it's available down to Python 2.4, which is used in Plone 2.5.

**Comment by thet** [comment]
looks good and it solves the problem of running tests only for specific plone versions.
regarding version comparisons, see my comment above.

**Pull request merged**